generated/nidaqmx/constants.py

```diff
@@ -609,33 +618,36 @@ class ResistanceConfiguration(Enum):
 
 
 class ResistanceUnits(Enum):
+    MILLI_OHMS = 10383
     OHMS = 10384  #: Ohms.
+    MEGA_OHMS = 10386
     FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.
     FROM_TEDS = 12516  #: Units defined by TEDS information associated with the channel.
 
 
 class ResistorState(Enum):
-    PULL_UP = 15950  #: pull up state for pull up/pull down resistors
-    PULL_DOWN = 15951  #: pull down state for pull up pull down resistors
+    PULL_UP = 15950  #: Pull up.
+    PULL_DOWN = 15951  #: Pull down.
 
 
 class ResolutionType(Enum):
+    DIGITS = 10154
     BITS = 10109  #: Bits.
 
 
 class SCXI1124Range(Enum):
     ZERO_TO_ONE_V = 14629
     ZERO_TO_FIVE_V = 14630
     ZERO_TO_TEN_V = 14631
-    NEG_1_TO_1_V = 14632
-    NEG_5_TO_5_V = 14633
+    NEG_1TO_1_V = 14632
+    NEG_5TO_5_V = 14633
     NEG_10_TO_10_V = 14634
     ZERO_TO_TWENTY_M_A = 14635
```

Comment:
Ugh both old and new are bad. If an attribute doesn't actually use this, lets delete it, But I suspect it does despite SCXI being EOL since `nidaqmx` works with old versions. Can you make these all consistent? I hate spelling out numbers, but we may not be able to lead with a number, so if that is the case them make them all spelled out. Also M_A is bad, make it MA.